src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -531,21 +537,28 @@ export default {
 		}
 
 		// Async data fetch for MARS-317 and share button
-		const { data } = await this.apollo.query({
-			query: mountedQuery,
-			variables: {
-				loanId: this.loanId,
-			},
-		});
-		const loan = data?.lend?.loan;
-		this.partnerName = loan?.partnerName ?? '';
-		this.partnerCountry = loan?.partner?.countries[0]?.name ?? '';
-		this.lender = data?.my?.userAccount ?? {};
+		if (this.enabledExperiment) {
```

Comment:
I don't think we should gate this entire query...We need the `lender` data for the share button.